src/NzbDrone.Core/Music/TrackService.cs

```diff
@@ -100,6 +102,43 @@ select new
             return matches.OrderByDescending(e => e.NormalizedLength).FirstOrDefault()?.Track;
         }
 
+        public Track FindTrackByTitleInexact(int artistId, int albumId, int mediumNumber, int trackNumber, string releaseTitle)
+        {
+            double fuzzThreshold = 0.6;
```

Comment:
Why different threshold for Track than Album?

Reply:
Simple answer is that was what's needed to get the test cases to pass.

I thought we could afford to be a bit less picky on the track name because we have an extra piece of information: the best track match has to have the right track number to be accepted but for albums we only have the name to go on.

For the examples that motivated this, the album name was generally pretty close (Eg it had CD appended) but track names can be a bit more off